ext/afform/core/ang/af/afForm.component.js

```diff
@@ -54,6 +54,18 @@
 
       this.submit = function submit() {
         var submission = crmApi4('Afform', 'submit', {name: ctrl.getFormMeta().name, args: $routeParams, values: data});
+        var metaData = ctrl.getFormMeta();
+        if (undefined != typeof metaData.redirect) {
```

Comment:
This works like `!empty()` in php:
```suggestion
        if (metaData.redirect) {
```

Reply:
Updated